app/build.gradle

```diff
@@ -116,6 +116,10 @@ android {
             applicationIdSuffix ".debug"
             versionNameSuffix "-debug"
             signingConfig signingConfigs.debug
+
+            buildConfigField "String", "IP_SIMPLE_API_ACCESS_KEY", "\"AIzaSyBhJJeiBWKTfbV446m2sgJCP-EogEWZUh0\""
```

Comment:
I actually added it when I work on AppInvite feature. That makes sense to move completely to that json file. 
Should I do it in this PR?